fast_bert/data_cls.py

```diff
@@ -222,7 +294,10 @@ def get_train_examples(
     ):
 
         if size == -1:
-            data_df = pd.read_csv(os.path.join(self.data_dir, filename))
+            print('>>>', os.path.join(self.data_dir, filename))
+            data_df = pd.read_csv(os.path.join(self.data_dir, filename), lineterminator='\n')
```

Comment:
A quoi sert le `lineterminator` ici?

Reply:
haha I recently learned that there exists another next line terminator called \r and some tweets have it which screws things over